### PR TITLE
Feature/add okx adapter

### PR DESCRIPTION
**Pull request by philbow61**
## Description

added the okx adapter 

## Other changes

removed error in the bitstamp adapter  unnecessary '/'

## Tested

- tested each of the function in the adapter
- ran oracle locally

## Related issues

- Fixes https://github.com/mento-protocol/mento-general/issues/120

### CODE REVIEW

**Comment by martinvol** [comment]
nit: you're also adding bitstramp, so great if you add it to the PR title as well

File: src/exchange_adapters/okx.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+   *    "volCcy24h":"1642445.37682",
+   *    "vol24h":"2177089.719932",
+   *    "ts":"1674479195109",
+   *    "sodUtc0":"0.685",
+   *    "sodUtc8":"0.698"
+   *   }
+   *  ]
+   * }
+   */
+
```
Comment (ninocomputer) [inline src/exchange_adapters/okx.ts:64]: nit: remove this line

File: src/exchange_adapters/okx.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    const tickerJson = await this.fetchFromApi(
+      ExchangeDataType.TICKER,
+      `market/ticker?instId=${this.pairSymbol}`
+    )
+
+    return this.parseTicker(tickerJson)
+  }
+
+  async fetchTrades(): Promise<Trade[]> {
+    // Trade data is not needed by oracle but is required by the parent class.
```
Comment (ninocomputer) [inline src/exchange_adapters/okx.ts:28]: maybe we should add an issue in this sprint to get rid of this across the codebase. the more adaptors we add the more annoying it's going to get

**Review by ninocomputer — COMMENTED** [review]
minor comments, otherwise lgtm 🚢

**Comment by ninocomputer** [comment]
> nit: you're also adding bitstramp, so great if you add it to the PR title as well

he's just making a drive-by change in bitstamp so I don't think we need to add it to the title

**Review by ninocomputer — APPROVED** [review]

File: src/exchange_adapters/okx.ts (hunk 2 above)
Comment (philbow61) [reply src/exchange_adapters/okx.ts:28]: agree

Comment (tobikuhlmann) [reply src/exchange_adapters/okx.ts:28]: ++

**Review by tobikuhlmann — APPROVED** [review]
- [x] tests look good and pass
- [x] runs locally and reports correct KuCoin pair prices

LGTM

**Pull request merged**